test/switch-s3-hosts.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+
+//
+// utility to switch s3 targets for local testing. if the s3 buckets are left
+// pointing to the mapbox-node-pre-gyp-public-testing-bucket and you don't have
+// write permissions to those buckets then the tests will fail. switching the
+// target allows the tests to be run locally (even though the CI tests will fail
+// if you are not a collaborator to the mapbox/node-pre-gyp repository).
+//
+// this replaces the mapbox-specific s3 URLs with an URL pointing to an S3
+// bucket which can be written to. each person using this will need to supply
+// their own `toLocal.target` and `toMapbox.source` values that refer to their
+// s3 buckets (and set AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY
+// appropriately).
+//
+// reset to the mapbox settings before committing.
+//
```

Comment:
This looks like a utility script rather than a test, correct? If so, and I'm understanding right, how about moving this to the `scripts/` directory?

Reply:
moved.